tests/test.login.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { test, expect } from "@playwright/test";
+import HomePage from '../pages/home.page';
+import ProfilePage from '../pages/profile.page';
+
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const validEmail: string = process.env.VALID_EMAIL || '';
+const validPassword: string = process.env.VALID_PASSWORD || '';
+const validPhone: string = process.env.VALID_PHONE || '';
+const homepageUrl: string = process.env.HOMEPAGE_URL || '';
+
+test.beforeEach(async ({ page }) => {
+    const homepage = new HomePage(page);
+    await homepage.navigate('/');
+    await homepage.clickOnEnterBtn();
+});
+
+test('test case C200: Authorization with empty fields', async( {page} ) => {
+    const homepage = new HomePage(page);
+
+    await homepage.clickOnSubmitLoginFormBtn();
+
+    await homepage.checkAutorizationFormIsDisplayed();
+    await expect(await homepage.checkInputError('email')).toBe(true);
+    await expect(await homepage.checkInputError('password')).toBe(true);
+
+    await homepage.fillInput('email', validEmail);
+    await homepage.clickOnSubmitLoginFormBtn();
+    await homepage.checkAutorizationFormIsDisplayed();
+    await expect(await homepage.checkInputError('email')).toBe(false);
+    await expect(await homepage.checkInputError('password')).toBe(true);
```

Comment:
Aren't you expecting one error message to appear and another not? why so complicated?